code/datums/components/storage/concrete/bag_of_holding.dm

```diff
@@ -22,6 +22,8 @@
 					M.visible_message("<span class='danger'>The bluespace collapse crushes the air towards it, pulling [M] towards the ground...</span>")
 					M.Knockdown(5, TRUE, TRUE)		//Overrides stun absorbs.
 			T.TerraformTurf(/turf/open/chasm/magic, /turf/open/chasm/magic)
+		for (var/obj/structure/ladder/unbreakable/binary/ladder in GLOB.ladders)
+			ladder.ActivateAlmonds()

```

Comment:
Why are you doing this to every ladder (of this kind) when not every ladder is necessarily associated with this?

Reply:
Pretty sure /binary ones are just singularity ones.

Reply:
There are multiple kinds of binary ladder

Reply:
all the /binary ladders are associated with boh